src/jit/jitconfigvalues.h

```diff
@@ -251,11 +233,25 @@ CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.
 #if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
 // Enable AVX instruction set for wide operations as default. When both AVX and SSE3_4 are set, we will use the most
 // capable instruction set available which will prefer AVX over SSE3/4.
-CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 1)
-#else  // !defined(_TARGET_AMD64_) && !defined(_TARGET_X86_)
+CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 1)             // Enable AVX
```

Comment:
nit: It might be nice to order this list. Either alphabetically, or some other "logical" order.

Reply:
Thanks, will fix.